extensions/amp-accordion/1.0/amp-accordion.css

```diff
@@ -15,9 +15,13 @@
  */
 
 /* heading/content */
-.i-amphtml-accordion-header,
+.i-amphtml-accordion-header {
+  margin: 0;
+}
+
 .i-amphtml-accordion-content {
   margin: 0;
+  display: block !important;
```

Comment:
I was wrong for this one and the one in the JSS file :( But we can't force `display: block` because `display: flex` is a perfectly valid value here. Everything else should work unchanged though. Let's just remove the forced `display: block`.

Reply:
Updated in latest revision.  (removed `!important`). Also cleaned up the `css` file a bit